app/lib/submission_builder/ty2024/states/md/documents/md1099_g.rb

```diff
@@ -0,0 +1,56 @@
+module SubmissionBuilder
+  module Ty2024
+    module States
+      module Md
+        module Documents
+          class Md1099G < SubmissionBuilder::Document
+            include SubmissionBuilder::FormattingMethods
+
+            def document
+              form1099g = @kwargs[:form1099g]
+              state_abbreviation = "MD"
+
+              build_xml_doc("MD1099G", documentId: "MD1099G-#{form1099g.id}") do |xml|
+                if form1099g.payer_name && form1099g.payer_name != ''
```

Comment:
`present?` should check if not nil and not empty string
```suggestion
                if form1099g.payer_name.present?

```